worker/orca_grader/tests/docker_images/test_docker_image_loading.py

```diff
@@ -20,8 +20,7 @@ def setUpClass(cls) -> None:
         return super().setUpClass()
 
     def test_image_download_and_loading(self):
-        retrieve_image_tgz_from_url(
-            "hello-world", "http://localhost:9000/images/hello-world.tgz")
+        retrieve_image_tgz_for_sha("hello-world")
```

Comment:
If you're going to rename things to be `..._sha`, then it seems quite odd to pass in a "sha" of `"hello-world"`.